src/continuous_aggregate/view_definitions.py

```diff
@@ -0,0 +1,653 @@
+VIEW_STRINGS = {
+    # NOTE: the regex in these queries cause a DeprecationWarning: invalid escape sequence
+    # For this reason we use a rawstring for these queries

```

Comment:
Kan je dit in aparte files zetten? Ik wil voorkomen dat we weer zo'n extreem groot view definitions file krijgen. Dus één file per view. Tenzij ze deel zijn van één migratie, dan is het misschien wel logisch om ze bij elkaar te zetten.